config.json

```
@@ -16,9 +16,15 @@
     "highlightjs_language": "bat"
   },
   "files": {
-    "solution": [],
-    "test": [],
-    "example": [],
+    "solution": [
+      "%{pascal_slug}.bat"
+    ],
+    "test": [
+      "%{pascal_slug}Test.bat"
+    ],
+    "example": [
+      ".meta/Example.bat"
```

Probably better to not capitalize the name of this file.

```suggestion
      ".meta/example.bat"
```

---

It's fine to capitalize as that is how all files are named in this track.